.github/workflows/examples.yml

```diff
@@ -10,7 +10,8 @@ jobs:
         os: [ubuntu-latest, windows-latest, macos-latest]
         example:
           - "examples/arduino-blink"
-          - "examples/arduino-external-libs"
+          - "examples/arduino-blink"
+          - "examples/arduino-blink-earlephilhower"
```

Comment:
Seems like there are two `arduino-blink` examples now and `arduino-external-libs` is removed.

Reply:
Accidental removal -- will be reverted.

Reply:
Should we combine the blink examples in one? I wanted to make clear to the reader that there are two possible cores that way and provide starting points for working with each core.

Reply:
Sure, let's combine them.

Reply:
Examples were combined and a note was added.